extensions/wikia/DiscussionModeration/DiscussionModeration.setup.php

```diff
@@ -1,21 +1,27 @@
 <?php
 
 $wgExtensionCredits['other'][] = array(
-	'name' => 'feeds-reported-page-api',
+	'name' => 'DiscussionModeration',
 	'author' => array(
 		'Fandom',
 	),
-	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia/FeedsReportedPage',
+	'url' => 'https://github.com/Wikia/app/tree/dev/extensions/wikia/DiscussionModeration',
 	'descriptionmsg' => 'feeds-reported-page-api-desc'
 );
 
 $dir = dirname(__FILE__) . '/';
 
 $wgAutoloadClasses['Post'] =  $dir . 'Post.class.php';
 $wgAutoloadClasses['PostBuilder'] =  $dir . 'PostBuilder.class.php';
-$wgAutoloadClasses['FeedsPermissionManager'] =  $dir . 'FeedsPermissionManager.class.php';
-$wgAutoloadClasses['FeedsReportedPageController'] =  $dir . 'FeedsReportedPageController.class.php';
-$wgAutoloadClasses['FeedsReportedPageGateway'] =  $dir . 'FeedsReportedPageGateway.class.php';
+$wgAutoloadClasses['DiscussionPermissionManager'] =  $dir . 'DiscussionPermissionManager.class.php';
+$wgAutoloadClasses['ReportedPostsHelper'] =  $dir . 'ReportedPostsHelper.class.php';
+$wgAutoloadClasses['DiscussionGateway'] =  $dir . 'DiscussionGateway.class.php';
+$wgAutoloadClasses['DiscussionModerationController'] =  $dir . 'DiscussionModerationController.class.php';
+$wgAutoloadClasses['FeedsReportedPageController'] =  $dir . 'DiscussionModerationController.class.php';
```

Comment:
Is it still valid? The point of the class alias was to unify the URL between app/ucp in order to use single code on feeds/d-f-e.

Reply:
there will be period of time when we deploy this change in app/ucp but feeds/d-f-e will use old controller name so we need to support both names. I'll remove alias after updating clients

Reply:
Makes sense, I just noticed the same thing in ucp 👍